### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by christopher-mohr**
PR for the next release (`1.1.0`).

We will take care of the full test dataset in the next release.

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/epitopeprediction branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/epitopeprediction)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [x] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/epitopeprediction/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Review by skrakau — CHANGES_REQUESTED** [review]
Looks good! Just a few minor changes and questions.

File: bin/epaa.py [hunk 1]
```diff
@@ -671,11 +698,32 @@ def get_matrix_max_score(allele, length):
...
+
+    else:
+        return np.nan
+
+
 def create_affinity_values(allele, length, j, method, max_scores, allele_strings):
     if not pd.isnull(j):
         if 'syf' in method:
-            return round(((100.0 / float(max_scores[allele_strings[('%s_%s' % (str(allele), length))]]) * float(j)) / 100.0) * 100, 2)
+            return max(0, round(((100.0 / float(max_scores[allele_strings[('%s_%s' % (str(allele), length))]]) * float(j)) / 100.0) * 100, 2))
```
[the remark this answers is not in the document]
Comment (christopher-mohr) [reply bin/epaa.py:720]: True 👍

File: docs/output.md [hunk 2]
```diff
@@ -47,3 +32,60 @@ HLHTHVYLF 9 17 3336962 ENSG00000127780 ENST00000248384 ENSP00000248384 SNP syfpe
 HTHVYLFLS 9 17 3336962 ENSG00000127780 ENST00000248384 ENSP00000248384 SNP syfpeithi-1.0 7.0 17.5 False False False c.173C>A p.Pro58His
 HVYLFLSNL 9 17 3336962 ENSG00000127780 ENST00000248384 ENSP00000248384 SNP syfpeithi-1.0 0.0 0.0 False False False c.173C>A p.Pro58His
 ```
+
+The prediction results are given as allele-specific score and affinity values per peptide. The computation of these values depends on the applied prediction method:
+
+* `Syfpeithi`:
+  * **Affinity**: Calculated as the half-max score of the corresponding matrix: `score(peptide) divided by the maximum score of the allele/length-specific matrix * 100`.
```
[the remark this answers is not in the document]
Comment (christopher-mohr) [reply docs/output.md:39]: Not sure if I understand this correctly but this line in `epaa.py` corresponds to the half-max score: 
`max_scores[allele_strings[('%s_%s' % (str(allele), length))]]`

The values are calculated by this function:
```
def get_matrix_max_score(allele, length):
    allele_model = "%s_%i" % (allele, length)
    try:
        pssm = getattr(__import__("Fred2.Data.pssms.syfpeithi.mat."+allele_model, fromlist=[allele_model]),
                           allele_model)
        return sum([max(scrs.values()) for pos, scrs in pssm.iteritems()])
    except:
        return np.nan
``` 

This isn't an affinity (or IC50).

**Review by skrakau — APPROVED** [review]
Looks great!

**Comment by skrakau** [comment]
The release date in the CHANGELOG should be updated when everything else is settled.

File: .github/workflows/awstest.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+          AWS_JOB_DEFINITION: ${{ secrets.AWS_JOB_DEFINITION }}
+          AWS_JOB_QUEUE: ${{ secrets.AWS_JOB_QUEUE }}
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+        run: |
+          aws batch submit-job \
+          --region eu-west-1 \
+          --job-name nf-core-epitopeprediction \
+          --job-queue $AWS_JOB_QUEUE \
+          --job-definition $AWS_JOB_DEFINITION \
+          --container-overrides '{"command": ["nf-core/epitopeprediction", "-r '"${GITHUB_SHA}"' -profile test --outdir s3://'"${AWS_S3_BUCKET}"'/epitopeprediction/results-'"${GITHUB_SHA}"' -w s3://'"${AWS_S3_BUCKET}"'/epitopeprediction/work-'"${GITHUB_SHA}"' -with-tower"], "environment": [{"name": "TOWER_ACCESS_TOKEN", "value": "'"$TOWER_ACCESS_TOKEN"'"}]}'epitopeprediction/work-'"${GITHUB_SHA}"' -with-tower"], "environment": [{"name": "TOWER_ACCESS_TOKEN", "value": "'"$TOWER_ACCESS_TOKEN"'"}]}'
```
Comment (ggabernet) [inline .github/workflows/awstest.yml]:
```suggestion
          --container-overrides '{"command": ["nf-core/epitopeprediction", "-r '"${GITHUB_SHA}"' -profile test --outdir s3://'"${AWS_S3_BUCKET}"'/epitopeprediction/results-'"${GITHUB_SHA}"' -w s3://'"${AWS_S3_BUCKET}"'/epitopeprediction/work-'"${GITHUB_SHA}"' -with-tower"], "environment": [{"name": "TOWER_ACCESS_TOKEN", "value": "'"$TOWER_ACCESS_TOKEN"'"}]}'
```

Comment (ggabernet) [reply .github/workflows/awstest.yml]: There is still this issue in the Awstest :)

File: README.md [hunk 4]
```diff
@@ -1,74 +1,65 @@
 # ![nf-core/epitopeprediction](docs/images/nf-core-epitopeprediction_logo.png)
 
-**A fully reproducible and state of the art epitope prediction pipeline.**.
-
-[![GitHub Actions](https://github.com/nf-core/epitopeprediction/workflows/nf-core%20CI/badge.svg)](https://github.com/nf-core/epitopeprediction/workflows/nf-core%20CI/badge.svg)
+[![GitHub Actions CI Status](https://github.com/nf-core/epitopeprediction/workflows/nf-core%20CI/badge.svg)](https://github.com/nf-core/epitopeprediction/actions)
+[![GitHub Actions Linting Status](https://github.com/nf-core/epitopeprediction/workflows/nf-core%20linting/badge.svg)](https://github.com/nf-core/epitopeprediction/actions)
 [![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A519.10.0-brightgreen.svg)](https://www.nextflow.io/)
+[![DOI](TODO)](todo)
```
Comment (ggabernet) [inline README.md]:
```suggestion
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3564666.svg)](https://doi.org/10.5281/zenodo.3564666)```

[the remark this answers is not in the document]
Comment (ggabernet) [reply README.md]: The DOI already exists https://doi.org/10.5281/zenodo.3564666

Comment (ggabernet) [inline README.md]:
```suggestion
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3564666.svg)](https://doi.org/10.5281/zenodo.3564666)
```

Comment (ggabernet) [reply README.md]: The badge already exists ;)

**Review by ggabernet — CHANGES_REQUESTED** [review]

**Review by ggabernet — APPROVED** [review]
Ah I don't know what happened that it was not updated. Looks good now!

**Comment by christopher-mohr** [comment]
Thanks @skrakau and @ggabernet!

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,8 +1,38 @@
 # nf-core/epitopeprediction: Changelog
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## nf-core/epitopeprediction V 1.0.0 - "Purple Nickel Shrimp"
+## v1.1.0 - 2020-10-09
```
Comment (christopher-mohr) [inline CHANGELOG.md:6]:
```suggestion
## v1.1.0 - Morgenstelle - 2020-10-20
```

**Pull request merged**